src/main/java/com/cinema/Main.java

```diff
@@ -17,10 +17,10 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.util.List;
-import lombok.extern.log4j.Log4j2;
+import org.jboss.logging.Logger;
```

Comment:
is this from a different logging dependency that is currently present in your pom.xml?